profile/README.md

```diff
@@ -23,34 +23,53 @@ The Data Lab team simultaneously contributes to childhood cancer research and to
 
 ## Single-cell Pediatric Cancer Atlas (ScPCA)
 
-[`The Single-cell Pediatric Cancer Atlas (ScPCA)`](https://scpca.alexslemonade.org/) focuses on single-cell and single-nuclei profiling with the goal of creating a publicly available atlas of pediatric cancer data. 
-Ten ScPCA awards were funded by Alex’s Lemonade Stand Foundation. 
-The ALSF-funded researchers submit their single-cell, single-nuclei, and bulk RNA sequencing data to the Data Lab for processing. 
-The data from these patient tumors are made available in one location through the [ScPCA Portal](https://scpca.alexslemonade.org/).
+ALSF created the [`Single-cell Pediatric Cancer Atlas (ScPCA)`](https://scpca.alexslemonade.org/) project to generate an unprecedented resource for the pediatric cancer research community. 
```

Comment:
This applies elsewhere, too – I don't think these project names need to be in back ticks. I know this is not new but is something I just noticed looking at the rich diff.

```suggestion
ALSF created the [Single-cell Pediatric Cancer Atlas (ScPCA)](https://scpca.alexslemonade.org/) project to generate an unprecedented resource for the pediatric cancer research community. 
```

Reply:
I can not easily comment on the refine.bio instance of this pattern.